client/src/hoc/__tests__/__snapshots__/layout.spec.js.snap

```diff
@@ -3,10 +3,73 @@
 exports[`<Layout /> should render component 1`] = `
 <div>
   <Header />
-  <div>
+  <div
+    style={
+      Object {
+        "minHeight": "95vh",
+      }
+    }
+  >
     <div>
       test
     </div>
   </div>
+  <footer
```

Comment:
I would suggest this component to be mocked. 
Makes no sense to know it's internal structure here as we are checking it's correctness in the test of the Footer component